### DISCUSSION TITLE
GPU (nvml)

### DISCUSSION THREAD

**Discussion by ghost** [sustainable-computing-io/kepler#1298] [General]
I cannot see the metrics for nvml in grafana for kepler.
What should I do exactly?

**Comment by marceloamaral** [comment]
@kazem26 which metrics are you looking for?

Few questions:
- Is your environment based on Bare-metal or VMs?
- Can you share the Kepler logs?
- I am assuming that you have access to GPUs in your cluster. Do you see `kepler_container_gpu_compute_util_total`?

**Comment by thisiskazem** [comment]
It's a bare-metal environment. there's a kubernetes cluster only with a master-node running and all the pods are running on the master-node. I've uploaded the logs of kepler here.
[metrics.pdf](https://github.com/sustainable-computing-io/kepler/files/14588323/metrics.pdf)

**Reply by rootfs** [reply]
from the metrics, there is no `kepler_node_gpu` or `kepler_container_gpu` metrics, have you enabled gpu in kepler container? i.e. `-enable-gpu=true`

**Reply by thisiskazem** [reply]
Yes. It is enabled

**Comment by thisiskazem** [comment]
I've installed kepler on a bare-metal server. My server has a graphic card installed. How should i check if the enable-gpu=true or not because I cannot see the metrics related to gpu.
